docs/how-to/rest-api-and-dynamodb.md

```diff
@@ -0,0 +1,1364 @@
+The table name, however, is static and won’t change; it’s safe to put it in the `.env` file.
+> 
+
+After doing that, we can start our REST API by running `npm run dev`.
+
+Let’s make sure that our *local* server is up and running by debugging it using cURL:
+
+```bash
+curl http://localhost:3000/animals/rin%20tin%20tin \
+    --data '{"breed":"German Shepard", "gender": "male"}' \
+    -XPOST -H 'Content-Type: application/json'| jq .
+```
+
+As you can see, *local* servers are hosted on the [`localhost`](http://localhost) domain or [`127.0.0.1`](http://127.0.0.1) IP address. It’s also common to see them running on the `3000` port, as being done in this server. (`8000`, `8080`, `5000` are also equally as common) 
+
+[Learn more about local hosting and ports.](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server)
+
+---
+
+Our database is currently empty, and it’s not very convenient to create API routes that fetch data when… well, there’s no data to fetch! That would make it nearly impossible to debug our RESTful API.
+
+Therefore, for the purposes of this tutorial, you can run the following script that generates some mock data to fill your database. Create a new file called `fill-db.js` and copy the contents of this script into it:
+
+```javascript
+import DynamoDb from "cyclic-dynamodb";
+
+import slugify from "slugify";
+import { v4 as uuidv4 } from "uuid";
+import { faker } from "@faker-js/faker";
+
+export const bikeAccessor = () => {
+  const type = faker.vehicle.bicycle();
+
+  const title = type + " " + faker.vehicle.model();
+  const handle = slugify(title).toLocaleLowerCase();
+
+  const minPrice = faker.datatype.number({
+    min: 800,
+    max: 3000,
+  });
+  const maxPrice = faker.datatype.number({
+    min: minPrice,
+    max: minPrice * 2,
+  });
+
+  return {
+    id: uuidv4(),
+    title,
+    handle,
+    productType: type,
+    createdAt: String(faker.datatype.datetime()),
+    description: faker.lorem.paragraph(),
+    vendor: faker.vehicle.manufacturer(),
+    availableForSale: faker.datatype.boolean(),
+    totalInventory: faker.datatype.number({ min: 1, max: 5 }),
+    priceRange: {
+      minPrice: {
+        amount: minPrice,
+        currencyCode: "USD",
+      },
+      maxPrice: {
+        amount: maxPrice,
+        currencyCode: "USD",
+      },
+    },
+  };
+};
+
+async function fillDataBaseWithBikes() {
+  const totalBikes = 25;
+  const randomBikes = Array(totalBikes).fill({}).map(bikeAccessor);
+
+  const db = DynamoDb(process.env.DYNAMO_DB);
+  const bikesCollection = db.collection("bikes");
+
+  let i = 1;
+  for (const bike of randomBikes) {
+    console.log(`Adding bike ${i}/${totalBikes}...`);
+    await bikesCollection.set(bike.id, bike);
```

Comment:
instead of awaiting each, just:
```
Promise.allSettled(randomBikes.map(async (bike) =>{
  console.log(`Adding bike ${i}/${totalBikes}...`);
  return bikesCollection.set(bike.id, bike);
})
```